board/safety.h

```diff
@@ -45,42 +45,43 @@ typedef struct {
   const safety_hooks *hooks;
 } safety_hook_config;
 
+// from cereal.car.CarParams.SafetyModel
 #define SAFETY_NOOUTPUT 0U
 #define SAFETY_HONDA 1U
 #define SAFETY_TOYOTA 2U
-#define SAFETY_GM 3U
-#define SAFETY_HONDA_BOSCH 4U
-#define SAFETY_FORD 5U
-#define SAFETY_CADILLAC 6U
-#define SAFETY_HYUNDAI 7U
-#define SAFETY_TESLA 8U
+#define SAFETY_ELM327 3U
+#define SAFETY_GM 4U
+#define SAFETY_HONDA_BOSCH 5U
+#define SAFETY_FORD 6U
+#define SAFETY_CADILLAC 7U
+#define SAFETY_HYUNDAI 8U
 #define SAFETY_CHRYSLER 9U
-#define SAFETY_SUBARU 10U
-#define SAFETY_GM_PASSIVE 11U
-#define SAFETY_MAZDA 12U
-#define SAFETY_GM_ASCM 0x1334U
-#define SAFETY_TOYOTA_IPAS 0x1335U
-#define SAFETY_ALLOUTPUT 0x1337U
-#define SAFETY_ELM327 0xE327U
+#define SAFETY_TESLA 10U
+#define SAFETY_SUBARU 11U
+#define SAFETY_GM_PASSIVE 12U
+#define SAFETY_MAZDA 13U
+#define SAFETY_TOYOTA_IPAS 16U
```

Comment:
Might we reserve the cereal-matching VW and Nissan safety numbers while you're in the neighborhood?

#define SAFETY_NISSAN 14U
#define SAFETY_VOLKSWAGEN 15U

A real Panda safety PR will be coming in the next few days, once I fix the last issue with VW torque rate limiting. Just tired of getting bumped off our safety number, this is our third bump. :)

Reply:
oh, no worries, it will not happen again. That's why 14 and 15 are not in there. 
The enumeration now matches what's in cereal. 
Just add:
```
#define SAFETY_VOLKSWAGEN 15U
```

whithin your PR